### ISSUE TITLE
Install only HIL client library. 

### ISSUE THREAD

**Issue by SahilTikale** [CCI-MOC/hil#1012]
We need to have a way to install only the HIL client. 
I am currently working on a scheduler [QUADS](https://github.com/SahilTikale/quads) which is a consumer of HIL. 
Currently I am copying the client folder from hil/client to QUADS /lib folder.

I think the correct way is to be able to install the hil library eg. `pip install hil_client_lib` or something like that. 
This will be helpful for other services that may interface with HIL or for people to install client library to remotely interface with HIL.

**Comment by zenhack** [comment]
I agree, I think we should split the client lib off into a separate
python package.

For the immediate term though, it's worth noting that it shouldn't break
anything to have the server stuff installed via pip; you can just not
use it -- if that makes your job easier..

Quoting Sahil Tikale (2018-05-09 12:12:11)

**Comment by naved001** [comment]
>  it's worth noting that it shouldn't break anything to have the server stuff installed via pip; you can just not use it -- if that makes your job easier..

yeah, `pip install git+https://github.com/cci-moc/hil` installs all hil libraries and then you can import your client library stuff.

**Comment by xuhang57** [comment]
I am a bit curious that, if we support installing the HIL client library only, does that mean we need to have an always-running hil server? How does that work?

I mean, people who just install the hil client still need to have a hil server so the client library could work, right?

**Comment by naved001** [comment]
@xuhang57
For instance, people would write some software that needs to talk to some
hil server and they use the the hil client library for that purpose. So
they can just put the hil client library in their requirements.

When that software is deployed it would need to point to some hil sever
that would be setup elsewhere. We don't need to have an always running hil
server.
